Libraries/LibWeb/SDLGamepadForward.h

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
suggestion: This header should be in the Gamepad folder since it's all about Gamepad :^)